src/index.html

```diff
@@ -22,6 +22,376 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#index.html"
+        class="header__logo-link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo image"
+          class="header__logo"
+        />
+      </a>
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a
+              href="#1"
+              class="header__nav-link is-active"
```

Comment:
Fix class everywhere
```suggestion
      <nav class="nav">
        <ul class="nav-list">
          <li class="nav-item">
            <a
              href="#1"
              class="nav-link is-active"
```

Reply:
@volodymyr-soltys97 соррі, що турбую, але в мене при такій зміні б'є помилку stylelint:  
39:13  error  Tag can't have more than one block  [nav-link]   (one-block)
 39:13  error  Tag can't have more than one block  [is-active]  (one-block)

2 errors
https://mate-academy.github.io/fed/bemlint/rules-description.html
BEMlint Failed